core-web/libs/sdk/experiments/src/lib/utils/DotLogger.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Logger for the dotCMS SDK
+ */
+export class DotLogger {
```

Comment:
Our loggin game globally is not the best, should we start thinking about using something more robust like: https://www.npmjs.com/package/winston

Reply:
Yes, I saw it. I like it.